src/pages/about-us/_OurLeadership.tsx

```diff
@@ -0,0 +1,351 @@
+    }
+    @media ${device.tablet} {
+        max-width: unset;
+        padding: 0;
+        grid-template-columns: ;
+    }
+`
+
+const StyledHeader = styled(Header)`
+    padding-bottom: 40px;
+    @media ${device.laptop} {
+        font-size: 28px;
+    }
+    @media ${device.tablet} {
+        padding-bottom: 24px;
+    }
+`
+const ModalFlex = styled(Flex)`
+    position: absolute;
+    top: 130px;
+    background-color: white;
+    z-index: 1;
+    border-radius: 5px;
+    box-shadow: rgba(0, 0, 0, 0.35) 0px 4px 15px;
+    white-space: pre;
+    width: fit-content;
+    @media ${device.tablet} {
+        top: 124px;
+    }
+    @media ${device.tabletS} {
+        top: 108px;
+    }
+    @media ${device.mobileL} {
+        top: 98px;
+    }
+`
+type StyledImageWrapperPropsType = ImageWrapperProps & {
+    onMouseOver: (event: FocusEvent<any>) => void
+    onMouseLeave: (event: FocusEvent<any>) => void
+    tabindex: string
+}
+const StyledImageWrapper = styled(ImageWrapper)<StyledImageWrapperPropsType>`
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    position: relative;
+    padding-bottom: 20px;
+    @media ${device.tabletS} {
+        width: 104px;
+        height: 124px;
+    }
+    
+    @media ${device.tabletL} {
+        &:nth-child(3n-2){
+            ${ModalFlex} { 
+                left: 0;
+            }
+        }
+        &:nth-child(3n){
+            ${ModalFlex} {
+                align-self:right;
+                right: 0;
+            }
+        }
+    }
+
+    @media ${device.mobileL} {
+        width: 88px;
+        height: 108px;
+    }
+    
+   
+}
+`
+
+const ModalHeader = styled(Header)`
+    font-size: 14px;
+    @media ${device.tabletL} {
+        font-size: 11px;
+    }
+`
+
+type ModalPropsType = {
+    name: string
+    position: string
+    link?: string
+}
+
+const StyledLogo = styled.img`
+    width: 32px;
+    height: 32px;
+    filter: grayscale(100%);
+    margin-bottom: 5px;
+    &:hover {
+        filter: unset;
+    }
+`
+
+const Modal = ({ name, position, link }: ModalPropsType) => {
+    return (
+        <ModalFlex ai="center" direction="column" width="unset" height="unset">
+            <ModalHeader type="unset" as="h4" padding="5px 5px 0" align="center">
+                {name}
+            </ModalHeader>
+            <Header
+                as="h4"
+                padding="0px 10px 5px"
+                type="sub-paragraph"
+                weight="normal"
+                align="center"
+            >
+                {position}
+            </Header>
+            {link && (
+                <LocalizedLink external="true" to={link} target="_blank" rel="noopener noreferrer">
+                    <StyledLogo src={Linkedin} alt="" />
+                </LocalizedLink>
+            )}
+        </ModalFlex>
+    )
+}
+type LeaderType = {
+    name: string
+    position: string
+    link: string
+    image: ImageDataLike
+}
+
+type LeaderPopsType = {
```

Comment:
```suggestion
type LeaderProps = {
```
Please follow the naming conventions on https://wikijs.deriv.cloud/e/en/Frontend/deriv-static/06-implementation-guide/ts-convention.

Reply:
we have  another one Leader type in the file, so I'll rename this one to "LeaderComponentType"

Reply:
! to LeaderProps